src/atoms/breadcrumbs/Breadcrumbs.html

```diff
@@ -1,41 +1,47 @@
-<div class="a-breadcrumbs">
-  <template v-for="(breadcrumb, i) in breadcrumbs">
-    <template v-if="breadcrumbs.length === i + 1">
-      <!-- @slot Current page (Scoped slot) -->
-      <slot
-        name="current"
-        :breadcrumb="breadcrumb"
-      >
-        <span
-          :key="`breadcrumb-${i}`"
-          :aria-label="breadcrumb.text"
-          class="a-breadcrumbs__item"
-          tabindex="0"
+<nav aria-label="Breadcrumb" class="a-breadcrumbs">
+  <ol class="a-breadcrumbs__list">
```

Comment:
Why you didn't use list component?

Reply:
I want to keep it as an atom - complete independent from the other components

Reply:
IMO `List` component can provide unnecessary complexity. So I vote for omitting that component.

Reply:
I loosed :D